src/lib/apiTypesV2/EntID.h

```diff
@@ -0,0 +1,59 @@
+#ifndef SRC_LIB_APITYPESV2_ENTID_H_
+#define SRC_LIB_APITYPESV2_ENTID_H_
+
+/*
+*
+* Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Fermín Galán Márquez
+*/
+#include <string>
+
+
+namespace ngsiv2
+{
+/* ****************************************************************************
+*
+* EntID -
```

Comment:
This class was originally in Subscription.h, but now it will be also used by Registration.h, so better to have it separated.

A side advantage of alinging registrations an subscriptions in NGSIv2 :)

NTC (just informative)